src/syft/lib/tensor/tensorbase.py

```diff
@@ -0,0 +1,179 @@
+# stdlib
+from functools import partial
+from typing import Any
+from typing import List
+from typing import Union
+
+# third party
+import numpy as np
+import torch
+
+# syft absolute
+from syft.decorators import syft_decorator
+from syft.lib.tensor.tensorbase_util import call_func_and_wrap_result
+from syft.lib.tensor.tensorbase_util import isfunc
+
+Num = Union[int, float]
+
+
+class ChildDelegatorTensor:
+    wrap_type: Any = None
+
+    def __getattr__(self, name: str) -> Any:
+        func_or_attr = getattr(self.child, name)
+        if isfunc(func_or_attr):
```

Comment:
Would `callable` replace isfunc? Also is there any need to some how create a dynamic wrapper for @property or `getset_descriptor` attributes?

Reply:
updated